### DISCUSSION TITLE
创建物化视图时，当聚合字段中，存在某个字段的值可能为空时，报DATA_QUALITY_ERROR

### DISCUSSION THREAD

**Discussion by GitHubChenLin** [apache/doris#21928] [A - General / Q&A]
如题：
版本：1.1
我尝试给表（聚合模型）创建物化视图时，当groupby字段中有某个字段的值为null或“”时，创建物化视图报DATA_QUALITY_ERROR；当我去掉这些“空字段”时，创建成功。

**Comment by yagagagaga** [comment]
If possible, please upgrade Doris version to latest.
